submissions/peterIyiola/html/productPage-1.html

```diff
@@ -1,89 +1,100 @@
    </head>
    <body>
-      <div>
-         
-            <h1><b>Better Buys</b></h1>
-          <a href="./productPage-1.html">Product page</a><br><br>
-         <a href="../htmlForms/contactForm.html">Contact Form</a><br>
-         <h4><i>Your neighbourhood electronics store</i></h4>
-         <hr>
-         <div>
-            <input type="text" name="box" placeholder="Type your Search here"</input>
-            <button>Search</button> <br>
-            <br>
-            <button>Signin</button>
-            <button>Register</button>
+      <div class="flex-container">
+         <div class="head">
+            <img src="../images/BBlogo.jpg" alt="BetterBuys Logo" height="40px" max-width="100%"/>
          </div>
-         <!--What we sell-->
-         <div>
-            <h3>Check us out for your:</h3>
-            <ul>
-               <li>TVs</li>
-               <li>Sound Systems</li>
-               </Sound>
-               <li>Air Conditioners</li>
-               <li>Refrigertors</li>
-            </ul>
+         <div class="productpagelink">
+            <a href="productPage-1.html">Product Page</a>
          </div>
-         <section>
-         <!--Product Images and Description-->
-         <div>
-            <ul>
-               <li>
-                  <a href="#"><img src="TVs.jpg" alt="Smart TV"></a>
-                  <p>65 inches QLED SMART TV 
-                     <br>
-                     +Wall Brackets and 12 Months warranty<br>
-                     Price = 250,000
-                  </p>
-               </li>
-               <li>
-                  <a href="#"><img src="HT.jpg" alt="Home Theatre" ></a>
-                  <p>Samsung Home Theatre System <br>
-                     Price=120,000 <br>
-                     High Definition Surround System
-                     Bluetooth Enabled
-                  </p>
-               </li>
-               <hr>
-               <p>Thermocool 2HP Air conditioner <br>
-                  Price=170,000
-               </p>
-               <hr>
-               <p>Hisense Regrigerators<br>
-                  Price =200,000 
-                  <br>
-                  Hisense 24Litres Refrigerator<br>
-                  Buttom Freezer and Water Dispenser
-               </p>
-            </ul>
+         <div class="contactlink">
+            <a href="../htmlForms/contactForm.html">Contact Us</a>
          </div>
-         <div>
-            <h4>
-               <p></p>
-               Similar Products
-               <ol>
-                  <li>LG LED 55 Inches ANdroid TV</li>
-                  <li>Hisense HD Sound System</li>
-                  <li>Polystar Double Door Refrigerator</li>
-                  <li>Scanfrost Air Purifier Air conditioner</li>
-               </ol>
-               </p>
-            </h4>
+         <div class="signuplink">
+            <a href="../htmlForms/Betterbuys.html">Sign Up</a>
```

Comment:
update the link here, the name has changed. @Iyiola-stack

Reply:
@Iyiola-stack The link has not been updated. You changed the name of the page???